online-shop/src/app/components/presentational/products-list-view/products-list-view.component.scss

```diff
@@ -0,0 +1,39 @@
+body {
```

Comment:
It seems that these classes can be applied to multiple components and elements. It would be a good practice to move them to the global `styles.scss` file

Reply:
moved them. you are right, many styles can be global in this project.